tests/integration/support/steps.go

```diff
@@ -170,6 +170,30 @@ func (c *ScenarioCtx) InTheResponseShouldMatch(argumentType string, value *gherk
 	default:
 		break
 	}
+
+	return fmt.Errorf("response for %s not equal", argumentType)
+}
+
+// InTheResponseShouldHaveThisExactlyTimes represents a step in the feature file
+func (c *ScenarioCtx) InTheResponseShouldHaveThisExactlyTimes(argumentType string, count int, value *gherkin.DocString) error {
+	switch argumentType {
+	case "listener_called":
```

Comment:
I am not sure, but all these strings should be constant and placed in constant.go
@mikeng13 do u suggest to have those strings in separate file?